From 3f05d555080522d7d0503e070a7c2dc28a7e4090 Mon Sep 17 00:00:00 2001
From: Kaarle Ritvanen <kunkku@alpinelinux.org>
Date: Wed, 17 Nov 2021 14:50:11 +0200
Subject: [PATCH 1/4] Revert "update-kernel: Make path for tmpdir changeable
 via cli flag"

This reverts commit 661f3b90f29f96a733aaff62ff0ca899ed34234d.
---
 update-kernel.in | 12 +++---------
 1 file changed, 3 insertions(+), 9 deletions(-)

diff --git a/update-kernel.in b/update-kernel.in
index fc78518..2d70035 100644
--- a/update-kernel.in
+++ b/update-kernel.in
@@ -31,7 +31,6 @@ REPOSITORIES_FILE=/etc/apk/repositories
 KEYS_DIR=/etc/apk/keys
 SIGNALS="HUP INT TERM"
 TMPDIR=
-TMPDIR_PATH="/tmp"
 features=
 modloopfw=
 
@@ -56,7 +55,6 @@ usage() {
 		         -F|--feature <feature>    Enable initfs feature
 		         -p|--package <package>    Additional module or firmware package
 		         -s|--modloopsign          Sign modloop with abuild key
-		         -t|--tmpdir <directory>   Choose different (non RAM) temporary directory for build files etc. 
 		         -v|--verbose              Verbose output
 		         -k|--apk-pubkey <key>     Include given key in initramfs
 		         -K|--hostkeys             Include host keys in initramfs
@@ -70,8 +68,8 @@ usage() {
 }
 
 QUIET_OPT="--quiet"
-OPTS=$(getopt -l arch:,build-dir:,flavor:,feature:,modloopfw:,help,package:,modloopsign,tmpdir:,verbose,apk-pubkey:,hostkeys,compression:,media,repositories-file:,keys-dir: \
-	-n $SCRIPT -o a:b:f:F:hp:st:vk:KC:Md: -- "$@") || usage 1
+OPTS=$(getopt -l arch:,build-dir:,flavor:,feature:,modloopfw:,help,package:,modloopsign,verbose,apk-pubkey:,hostkeys,compression:,media,repositories-file:,keys-dir: \
+	-n $SCRIPT -o a:b:f:F:hp:svk:KC:Md: -- "$@") || usage 1
 
 eval set -- "$OPTS"
 while :; do
@@ -107,10 +105,6 @@ while :; do
 	-s|--modloopsign)
 		MODLOOPSIGN=1
 		;;
-	-t|--tmpdir)
-		shift
-		TMPDIR_PATH="$1"
-		;;
 	-v|--verbose)
 		QUIET_OPT=
 		;;
@@ -224,7 +218,7 @@ fi
 
 [ "$ARCH" ] || ARCH=$(apk --print-arch)
 
-TMPDIR=$(mktemp -d "$TMPDIR_PATH/$SCRIPT.XXXXXX")
+TMPDIR=$(mktemp -d /tmp/$SCRIPT.XXXXXX)
 ROOT=$TMPDIR/root
 BOOT=$ROOT/boot
 
-- 
2.33.1

